doc/Type/Buf.pod6

```diff
@@ -90,7 +90,7 @@ Invokes the C<subbuf-rw> method on the specified C<Buf>:
 
 =head2 method reallocate
 
-    method reallocate($elems)
+    method reallocate(Buf:D: Int:D $elems)
```

Comment:
In this case it might correspond to the actual definition in the source, but in that case you need to explain that reallocate will throw if it's not defined.

Reply:
I don't think we need to explicitly call out that `Buf.reallocate` will throw a type error if called with an undefined `Buf` for two reasons:  First, that's just what it _means_ to require a defined invocant, so mentioning it again would be redundant.  Second, it's inconsistent with how we handle defined invocants everywhere else – we never call out that they throw a type error for undefined invocants.  (Some random examples: [Pair.invert](https://docs.raku.org/type/Pair#method_invert), [Capture.list](https://docs.raku.org/type/Capture#method_list), [Baggy.hash](https://docs.raku.org/type/Baggy#method_hash), [Real.sign](https://docs.raku.org/type/Baggy#method_hash))